apache/tests/compose/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM httpd:2.4.12
```

Comment:
Since you figured out the issue can we remove this file and do everything in the compose file?

Reply:
No, I'm still using this. Using a file as a volume was giving me an error on gitlab

Reply:
Hmm, it's working elsewhere https://github.com/DataDog/integrations-core/blob/master/redisdb/tests/compose/standalone.compose

Reply:
Yea, I don't really understand why it gave me an error

Reply:
I'll try it